storage/entry.go

```diff
@@ -276,6 +283,10 @@ func (s *Storage) RefreshFeedEntries(userID, feedID int64, entries model.Entries
 			}
 		} else {
 			err = s.createEntry(tx, entry)
+			if intg != nil && err == nil {
+				localEntry := entry
+				integration.PushEntry(localEntry, intg)
```

Comment:
This part should probably go in `processor/ProcessFeedEntries()` instead of the storage package. Some enhancements there might be required.

Also, notifications should be send in a Goroutine.